Makefile

```diff
@@ -1,18 +1,8 @@
 CLEANUP_FILES = charmcraft.yaml
 
-.PHONY: all ubuntu-advantage ubuntu-advantage_noble ubuntu-pro ubuntu-pro_noble clean
+.PHONY: all ubuntu-pro ubuntu-pro_noble clean
 
-all: ubuntu-advantage ubuntu-advantage_noble ubuntu-pro ubuntu-pro_noble
```

Comment:
Maybe keep a `legacy` target as well, with the old charms?

We may still need to build them, just not as part of the regular process.